tests/FSharp.Compiler.ComponentTests/EmittedIL/Structure/UnionTypeWithSignature02.fs

```diff
@@ -2,7 +2,7 @@
 // Regression test for FSHARP1.0:4040
 // "Signature files do not prevent compiler-generated public constructors from leaking out of discriminated unions"
 // Note that the corresponsing .fsi file is NOT missing the "| C of int" part of the DU
-namespace N
+module N
```

Comment:
Why this change?

Reply:
Because the test was never executed (see last point of PR description) and wrong.